src/servers/main.cc

```diff
@@ -295,6 +297,27 @@ std::vector<Option> options_
        "Name of the model to be loaded on server startup. It may be specified "
        "multiple times to add multiple models. Note that this option will only "
        "take affect if --model-control-mode=explicit is true."},
+      {OPTION_RATE_LIMIT, "rate-limit", Option::ArgStr,
+       "Specify the mode for rate limiting. Options are \"off\" and "
+       "\"execution_count\". The default is \"off\". "
```

Comment:
I think the default should be "execution_count" so that rate limiting works out-of-the-box